pkgs/applications/editors/jetbrains/versions.json

```diff
@@ -18,6 +18,15 @@
       "version-major-minor": "2022.1.1",
       "build_number": "222.3345.126"
     },
+    "gateway": {
+      "update-channel": "Gateway EAP",
+      "url-template": "https://download.jetbrains.com/idea/gateway/JetBrainsGateway-{version}.tar.gz",
+      "version": "2022.3",
+      "sha256": "031a9dbef0929b2fac01decf803f385b040a623a5c7bb70f066aec7fa4afa768",
+      "url": "https://download.jetbrains.com/idea/gateway/JetBrainsGateway-223.6160.27.tar.gz",
+      "version-major-minor": "2022.3",
```

Comment:
Gateway has been updated since this PR was created, rebase this PR to current master and run update.py again